### PR TITLE
Feature/30646 - Table element

### PR DESCRIPTION
**Pull request by kjugi**
+ Add basic table element
+ Add customising each element in config
+ Link tab in styles
+ Add `--odd-even` modifier
+ RWD ready table

https://dev.snowdog.pro/issues/30646

### CODE REVIEW

File: components/02-elements/table/_table.scss [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    background: $white;
+    margin: 1em 0;
+    border: 1px solid $gray-light;
+    border-radius: $table-radius;
+    color: $black;
+    border-collapse: separate;
+    border-spacing: 0;
+    font-size: $font-size-small;
+
+    &__odd-even {
```
Comment (Igloczek) [inline components/02-elements/table/_table.scss]: Modifier? Not used anywhere.

File: components/02-elements/table/table.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<table class="table {{ additionalClasses }}" {{{ additionalAttributes }}}>
```
Comment (Igloczek) [inline components/02-elements/table/table.hbs]: Rename `additionalClasses `  to `class`

Comment (kjugi) [inline components/02-elements/table/table.hbs]: what about other long names? Cut it too?

File: components/02-elements/table/_table.scss (hunk 1 above)
Comment (kjugi) [inline components/02-elements/table/_table.scss]: Preview created

**Pull request merged**